index.bs

```diff
@@ -162,21 +162,26 @@ This specification expresses a device's physical orientation as a series of rota
 
 The sequence of rotation steps is a set of intrinsic Tait-Bryan angles of type Z - X' - Y'' ([[EULERANGLES]]) that are applied on the <a>device coordinate system</a> defined in [[!ACCELEROMETER]] and summarized below:
 
-* x is in the plane of the screen or keyboard and is positive towards the right hand side of the screen or keyboard.
-* y is in the plane of the screen or keyboard and is positive towards the top of the screen or keyboard.
-* z is perpendicular to the screen or keyboard, positive out of the screen or keyboard.
+* <dfn for="Device Orientation">x</dfn> is in the plane of the screen or keyboard and is positive towards the right hand side of the screen or keyboard.
```

Comment:
Should this use the full name of the specification or the short name, i.e. "orientation-event". I guess it doesn't matter if these definitions aren't exported. It just looks a little weird.

Reply:
Right, I'll lower case them... this is just associating them with the concept of "device orientation", not so much the spec.

Reply:
Actually, never mind... Bikeshed needs those to be upper case. Repec doesn't require case matching on concepts.